### ISSUE TITLE
Can I join the organization?

### ISSUE THREAD

**Issue by TheOnlyCreator** [TARDIX/Kernel#3]
I would love to help you guys. If you need some proof of my worthiness, [here](https://github.com/TheOnlyCreator?tab=repositories) it is. My CC account is called Creator.

Thanks in advance

**Comment by DemHydraz** [comment]
Hello, @TheOnlyCreator. Thank you for showing interest in this project! (It is truly heartwarming.)

While it is interesting to see your previous work, it is not possible to accept you directly into the organization, atleast, not at this moment.

It would be better if you, instead, contribuited via pull requests. That way, we can see how you'd contribuite to the kernel in general.

If you want, the _low-hanging fruit_ at the moment of writing is the virtual file system. It should support creation of file (_inode_) tables and mounting of those to normal directories.


To summarize, currently, it is not possible for you to join the organization. However, if you do contribuite to the kernel via pull requests, and they are of good quality (as I can see your other work is.), the team will consider accepting you.

Respectfully, The TARDIX Team.

**Comment by TheOnlyCreator** [comment]
Thank you a lot for the fast response. I'll see what I can do.

**Comment by DemHydraz** [comment]
If you want to participate, indirectlly, in the development, we have a [Gitter chat room](https://gitter.im/TARDIX/Dev?utm_source=share-link&utm_medium=link&utm_campaign=share-link).

**Comment by RainbowDashDC** [comment]
Also, as another side comment: before anyone can join the org it must be brought up in the TC meetings.
